sim/hunter/marksmanship/chimera_shot.go

```diff
@@ -20,31 +20,27 @@ func (hunter *Hunter) registerChimeraShotSpell() {
 		ProcMask:    core.ProcMaskRangedSpecial,
 		Flags:       core.SpellFlagMeleeMetrics | core.SpellFlagAPL,
 
-		ManaCost: core.ManaCostOptions{
-			BaseCost: 0.12,
-			Multiplier: 1 -
-				0.03*float64(hunter.Talents.Efficiency) -
-				0.05*float64(hunter.Talents.MasterMarksman),
+		FocusCost: core.FocusCostOptions{
+			Cost: 50 - (float64(hunter.Talents.Efficiency) * 2),
 		},
 		Cast: core.CastConfig{
 			DefaultCast: core.Cast{
-				GCD: core.GCDDefault,
+				GCD: time.Second,
 			},
 			IgnoreHaste: true, // Hunter GCD is locked at 1.5s
 			CD: core.Cooldown{
 				Timer:    hunter.NewTimer(),
-				Duration: time.Second*10 - core.TernaryDuration(hunter.HasMajorGlyph(proto.HunterMajorGlyph_GlyphOfChimeraShot), time.Second*1, 0),
+				Duration: time.Second*10 - core.TernaryDuration(hunter.HasPrimeGlyph(proto.HunterPrimeGlyph_GlyphOfChimeraShot), time.Second*1, 0),
 			},
 		},
 
-		DamageMultiplier: 1 * hunter.markedForDeathMultiplier(),
-		CritMultiplier:   hunter.critMultiplier(true, true, false),
+		DamageMultiplier: 1,
+		CritMultiplier:   hunter.CritMultiplier(true, true, false),
 		ThreatMultiplier: 1,
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
 			baseDamage := 0.2*spell.RangedAttackPower(target) +
 				hunter.AutoAttacks.Ranged().BaseDamage(sim) +
-				hunter.AmmoDamageBonus +
 				spell.BonusWeaponDamage()
 			baseDamage *= 1.25
```

Comment:
this `1.25` multiplier can be moved to the static spell DamageMultiplier instead of here

Reply:
Removed. Like I wrote in the comment below, I don't think Chimera has the effects in cata anymore. Adjusted it.